Homework_8/Task_2.py

```diff
@@ -0,0 +1,12 @@
+
+params = {'v': 'some_id', 'list': 'some_list', 'index': '6', 't': '0s'}
+initial_str = 'https://www.youtube.com/watch?'
+query_string = ''
+
+for key, value in params.items():
+    query_string += f'{key}={value}&'
+
+query_string = query_string[:-1]
```

Comment:
query_string = query_string.strip('&')

Reply:
![image](https://github.com/vithaiduchyk/hillel_python_basic/assets/51053483/47ca6aca-503f-4d4e-b499-484ad8adf2a0)

Reply:
fixed and added to new pull request